2024-12-08 09:03:59,935:DEBUG:certbot._internal.main:certbot version: 3.0.1
2024-12-08 09:03:59,935:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-12-08 09:03:59,935:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/var/www/html', '--non-interactive', '--email', 'tbongkim03@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'parkingissue.online']
2024-12-08 09:03:59,935:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-12-08 09:03:59,952:DEBUG:certbot._internal.log:Root logging level set at 30
2024-12-08 09:03:59,953:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-12-08 09:03:59,954:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A separate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x72bb67d72600>
Prep: True
2024-12-08 09:03:59,956:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x72bb67d72600> and installer None
2024-12-08 09:03:59,956:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-12-08 09:04:00,067:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/2100901597', new_authzr_uri=None, terms_of_service=None), c809cf84a89ab438984ee845a16fa1ea, Meta(creation_dt=datetime.datetime(2024, 12, 8, 8, 56, 1, tzinfo=<UTC>), creation_host='729df04b4f14', register_to_eff=None))>
2024-12-08 09:04:00,071:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-12-08 09:04:00,074:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-12-08 09:04:00,599:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 746
2024-12-08 09:04:00,600:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 09:04:00 GMT
Content-Type: application/json
Content-Length: 746
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "mJA04Wd8CxU": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2024-12-08 09:04:00,601:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for parkingissue.online
2024-12-08 09:04:00,608:DEBUG:acme.client:Requesting fresh nonce
2024-12-08 09:04:00,608:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-12-08 09:04:00,780:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-12-08 09:04:00,781:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 09:04:00 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: MMy3dIC6VFuBK_V-KLwP5tPqAeeQ7XGetrMgfr1S2n1XlH4MCfI
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-12-08 09:04:00,781:DEBUG:acme.client:Storing nonce: MMy3dIC6VFuBK_V-KLwP5tPqAeeQ7XGetrMgfr1S2n1XlH4MCfI
2024-12-08 09:04:00,781:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "parkingissue.online"\n    }\n  ]\n}'
2024-12-08 09:04:00,783:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjEwMDkwMTU5NyIsICJub25jZSI6ICJNTXkzZElDNlZGdUJLX1YtS0x3UDV0UHFBZWVRN1hHZXRyTWdmcjFTMm4xWGxINE1DZkkiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "HyvjhWUPwgCNFB0ikIYqlZazTyMvfSDFb4ni15e9wB9eIoPwh_KE3MHIcsV59_D6qmtVk5vvOO_6us_jyAQrHyUmpW313exsnRfvqYflc1PgGAUqJpUmcmFzqTZPYspgKOiLgGfDcov-fz5PQlL19AONixbA16l60lU57kUJ-vSkFYrsFTDAVAHL6K9B13K9q0bSGRC1n6L4phh9nwhtBGnEZ0QkMrYpaCqqswS1jfeAYZfb2xb1_YzIXbWWVjz4D2o4hrJRPMcQYj_S7OSecxONxlSdDtSU2TuLnTEyDd5t6v56TfOWOO-nfy_sP5FMJyRiEvN9QU0qo7hDnhmanQ",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInBhcmtpbmdpc3N1ZS5vbmxpbmUiCiAgICB9CiAgXQp9"
}
2024-12-08 09:04:01,000:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 353
2024-12-08 09:04:01,000:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sun, 08 Dec 2024 09:04:00 GMT
Content-Type: application/json
Content-Length: 353
Connection: keep-alive
Boulder-Requester: 2100901597
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/2100901597/331074511667
Replay-Nonce: MMy3dIC6gg_FJSDm3fXoUMXa_lw18WYnWmKEYzorfz1rVDr_bIg
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-12-15T09:04:00Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "parkingissue.online"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz/2100901597/441692382167"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/2100901597/331074511667"
}
2024-12-08 09:04:01,001:DEBUG:acme.client:Storing nonce: MMy3dIC6gg_FJSDm3fXoUMXa_lw18WYnWmKEYzorfz1rVDr_bIg
2024-12-08 09:04:01,001:DEBUG:acme.client:JWS payload:
b''
2024-12-08 09:04:01,002:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2100901597/441692382167:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjEwMDkwMTU5NyIsICJub25jZSI6ICJNTXkzZElDNmdnX0ZKU0RtM2ZYb1VNWGFfbHcxOFdZbldtS0VZem9yZnoxclZEcl9iSWciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIxMDA5MDE1OTcvNDQxNjkyMzgyMTY3In0",
  "signature": "J7zQXCzirRmJPdNu-aHsDeXYZGmkfwYlHCDQCRmpmoJIQd70ZaXRkyFiBntWtOL4un7iEFODBlA1inuJjjt7gZ5OnE2RKfuR2-rQVj22uUPoNoYHptslpkPQZH_JoPj9Rvwo4_AzJopMQFdZvUBnHyG8-5t6Hxr4jFI7nt9UrPPKV_XAq8ZbU3IKWj5UntWxowXkrojrj3t-R3Q7_j78uml09vC-Tvc-KzZeA7lOAwSCORhKrsf2LSChXuKKermAU48A-_SIMlTMOpo2fH3VvaHOkIJxOPNGVrRrYRKIg9-77wnRmh2bQ45utFd_EtCEH6xz742c9Vwm8VBMzJ1VrA",
  "payload": ""
}
2024-12-08 09:04:01,179:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2100901597/441692382167 HTTP/1.1" 200 827
2024-12-08 09:04:01,179:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 09:04:01 GMT
Content-Type: application/json
Content-Length: 827
Connection: keep-alive
Boulder-Requester: 2100901597
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: qM6CNfq-iK3kOft3ETKxP12kzh6OjsxQWmG2BwDgf4wWfBu1xtc
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "parkingissue.online"
  },
  "status": "pending",
  "expires": "2024-12-15T09:04:00Z",
  "challenges": [
    {
      "type": "tls-alpn-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100901597/441692382167/7wctGw",
      "status": "pending",
      "token": "U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0"
    },
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100901597/441692382167/THCdIg",
      "status": "pending",
      "token": "U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0"
    },
    {
      "type": "dns-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100901597/441692382167/kphvyQ",
      "status": "pending",
      "token": "U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0"
    }
  ]
}
2024-12-08 09:04:01,179:DEBUG:acme.client:Storing nonce: qM6CNfq-iK3kOft3ETKxP12kzh6OjsxQWmG2BwDgf4wWfBu1xtc
2024-12-08 09:04:01,180:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-12-08 09:04:01,180:INFO:certbot._internal.auth_handler:http-01 challenge for parkingissue.online
2024-12-08 09:04:01,181:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/html for all unmatched domains.
2024-12-08 09:04:01,181:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/html/.well-known/acme-challenge
2024-12-08 09:04:01,182:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/html/.well-known/acme-challenge/U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0
2024-12-08 09:04:01,183:DEBUG:acme.client:JWS payload:
b'{}'
2024-12-08 09:04:01,186:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall/2100901597/441692382167/THCdIg:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjEwMDkwMTU5NyIsICJub25jZSI6ICJxTTZDTmZxLWlLM2tPZnQzRVRLeFAxMmt6aDZPanN4UVdtRzJCd0RnZjR3V2ZCdTF4dGMiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzIxMDA5MDE1OTcvNDQxNjkyMzgyMTY3L1RIQ2RJZyJ9",
  "signature": "kbwkWA1-exMPZnI21fiyuZ2l4lWopmH8AE5AytDn50DPMbW9bbQhS4uxTQ0m4yXQ6vLy8DtKgwyTSWwRfwfwzEvh7-a9UbetxhnLISPyC_vuzEoXBRFjXr36eV2UwIVTVOKMediTWSgR1d5gsW9Z7SQkZs6NIKMlJCyFmQXwx_KlkspUeUXUNifH-Fx0Hb88L4jYYCIH2Y-2zDftCrmGEv_pE4PKznBHrWXYHCnc-pEq3MDJ9lrI5Si0hbMumV8wafgyFPInxjaBOOnty4kSIBixpSla4wkfLqNQIoVXiz4KzPpd0sNXLrSDskUpXL5WUop43hMAw_F2FE0Jk4Rzuw",
  "payload": "e30"
}
2024-12-08 09:04:01,365:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall/2100901597/441692382167/THCdIg HTTP/1.1" 200 195
2024-12-08 09:04:01,365:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 09:04:01 GMT
Content-Type: application/json
Content-Length: 195
Connection: keep-alive
Boulder-Requester: 2100901597
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz/2100901597/441692382167>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall/2100901597/441692382167/THCdIg
Replay-Nonce: qM6CNfq-uCKYv2geEnWXE5s_5hGFGALsQp4lWBiNpW5JzIZ0MLg
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100901597/441692382167/THCdIg",
  "status": "pending",
  "token": "U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0"
}
2024-12-08 09:04:01,366:DEBUG:acme.client:Storing nonce: qM6CNfq-uCKYv2geEnWXE5s_5hGFGALsQp4lWBiNpW5JzIZ0MLg
2024-12-08 09:04:01,367:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-12-08 09:04:02,368:DEBUG:acme.client:JWS payload:
b''
2024-12-08 09:04:02,370:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2100901597/441692382167:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjEwMDkwMTU5NyIsICJub25jZSI6ICJxTTZDTmZxLXVDS1l2MmdlRW5XWEU1c181aEdGR0FMc1FwNGxXQmlOcFc1SnpJWjBNTGciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIxMDA5MDE1OTcvNDQxNjkyMzgyMTY3In0",
  "signature": "A1WrWS5_BL6InpG2DoIYbjUjFY6xVSUP_xtq0pyUuVYOtzSwmrW6cvlYcgUA4Tg0mP2yCpZ6oVKJFaPHdrBiGawtfwShCp0b62mgikZOOwaiAOHr0-I27Iq-H_IxJs6qEEqMhisw0ZhOpCFaarMkVOnYfEfWl8ykqpTUOTls-scVyDu4_QnJUeKmC3d-5UP6keVoXvFWS9qleGsAdzkIePnqFv_3SGeBxdn1GcR__zGfksuMiC0EOixwZVDd0jsazALqHLid2_irgtckITFwMJPrQX2jlxpGTQAmDMYU0tILIdoURUZ_teQlf-T0JqZrAoSlAANl2T5vqtkq3LpAxw",
  "payload": ""
}
2024-12-08 09:04:02,551:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2100901597/441692382167 HTTP/1.1" 200 1046
2024-12-08 09:04:02,552:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 09:04:02 GMT
Content-Type: application/json
Content-Length: 1046
Connection: keep-alive
Boulder-Requester: 2100901597
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: MMy3dIC6TVx6Uv5dJVH4uQX84RhlwjPqxrnCgbcsVCeUxz6D2YM
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "parkingissue.online"
  },
  "status": "invalid",
  "expires": "2024-12-15T09:04:00Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100901597/441692382167/THCdIg",
      "status": "invalid",
      "validated": "2024-12-08T09:04:01Z",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "3.37.20.117: Fetching http://parkingissue.online/.well-known/acme-challenge/U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0: Connection refused",
        "status": 400
      },
      "token": "U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0",
      "validationRecord": [
        {
          "url": "http://parkingissue.online/.well-known/acme-challenge/U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0",
          "hostname": "parkingissue.online",
          "port": "80",
          "addressesResolved": [
            "3.37.20.117"
          ],
          "addressUsed": "3.37.20.117"
        }
      ]
    }
  ]
}
2024-12-08 09:04:02,552:DEBUG:acme.client:Storing nonce: MMy3dIC6TVx6Uv5dJVH4uQX84RhlwjPqxrnCgbcsVCeUxz6D2YM
2024-12-08 09:04:02,552:INFO:certbot._internal.auth_handler:Challenge failed for domain parkingissue.online
2024-12-08 09:04:02,552:INFO:certbot._internal.auth_handler:http-01 challenge for parkingissue.online
2024-12-08 09:04:02,552:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: parkingissue.online
  Type:   connection
  Detail: 3.37.20.117: Fetching http://parkingissue.online/.well-known/acme-challenge/U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0: Connection refused

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-12-08 09:04:02,554:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-12-08 09:04:02,554:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-12-08 09:04:02,554:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-12-08 09:04:02,555:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/html/.well-known/acme-challenge/U1O7A1UBjz2mFpm8UzPV0HfhDfQSiuBGkjjmftpMZH0
2024-12-08 09:04:02,555:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-12-08 09:04:02,556:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1876, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1578, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 142, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-12-08 09:04:02,559:ERROR:certbot._internal.log:Some challenges have failed.
